DNN Platform/Website/admin/Skins/BreadCrumb.ascx.cs

```diff
@@ -175,18 +175,19 @@ protected override void OnLoad(EventArgs e)
                 // Begin breadcrumb
                 this._breadcrumb.Append("<span itemprop=\"itemListElement\" itemscope itemtype=\"http://schema.org/ListItem\">");
```

Comment:
this lne looks obsolete to me, or am I wrong?

Reply:
so true, will revoke the pull request and do it again.

Reply:
you may just fix it and resubmit your PR :)

Reply:
Do I need to resubmit the PR? I pushed the 2nd commit and it seems it was added automatically. But as this my very first PR I could be wrong.